### PR TITLE
Add partsegdata recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
@Czaki do you agree with being added as a maintainer?

**Comment by Czaki** [comment]
ok

File: recipes/partsegdata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+about:
+  home: https://4dnucleome.cent.uw.edu.pl/PartSeg/
+  summary: PartSeg data files
+  license: BSD-3-Clause
+  license_file: License.txt
+
+extra:
+  recipe-maintainers:
+    - conda-forge/napari
```
Comment (jaimergp) [inline recipes/partsegdata/meta.yaml:41]: This one does not depend on napari, but I assume is a dependency of a plugin. Do we still want the team to be maintaining this one too?

Comment (jaimergp) [reply recipes/partsegdata/meta.yaml:41]: Ok, fine, I'll leave it if it's a split then.

Comment (jaimergp) [reply recipes/partsegdata/meta.yaml:41]: Hm, come to think about it, if they are so intimately related, maybe a multi-output recipe is better suited... Would these three packages be updated at the same time or do they follow different release strategies @czaki?

Comment (Czaki) [reply recipes/partsegdata/meta.yaml:41]: No. PartSeg is updated much more often.  PartSegData PartSegCore-compiled-backend are updated rarely, only if needed.

Comment (jaimergp) [reply recipes/partsegdata/meta.yaml:41]: Then separate feedstocks it is. Thanks!

**Pull request merged**